src/pages/index.js

```diff
@@ -1,192 +1,224 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
 import Button from '@material-ui/core/Button';
-import Divider from '@material-ui/core/Divider';
 import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
 
 import SEO from '../components/Seo';
 import Layout from '../components/Layout';
 import Link from '../components/Link';
 
 import doRedirect from '../hoc/doRedirect';
 
+const useStyles = makeStyles(theme => ({
+  module: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    paddingBottom: '160px',
+  },
+  module__title: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingBottom: '56px',
+  },
+  module__title_picto: {
+    width: '64px',
```

Comment:
This does look like `drawerWidth` value, doesn't it?

Reply:
done